### PR TITLE
Fix translation checks in CI

### LINKED ISSUE #5019: [Automatic issue] Missing `_.trans()`.
It looks like one of our test cron detected missing translations.
You can see the latest output [here](https://github.com/napari/napari/actions/workflows/test_translations.yml).
There are likely new strings to either ignore, or to internationalise.

You can also Update the cron script to update this issue with better information as well.

Note that this issue will be automatically updated if kept open, or a new one will be created when necessary, if no open
issue is found and new `_.trans` call are missing.

### PR DESCRIPTION
**Pull request by kcpevey**
# Description

The scheduled translation check on CI has been failing for 2 years. This PR attempts to fix it by bringing everything back up to date - fixing translations from changes to the napari codebase and changes to pydantic.

As far as I know, the mechanics are not in place to _actually_ translate anything (https://github.com/napari/napari/issues/2195), but this check was put in place as a first step towards it. Keeping it up to date prior to release is recommended. 

## Type of change

- [x] Bug-fix (non-breaking change which fixes an issue)

# References

Resolves https://github.com/napari/napari/issues/5019
Related (maybe resolves): https://github.com/napari/napari/issues/3642

# How has this been tested?

- [x] example: all tests pass with my change

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality

## TODO list:
- [x] Need to temporarily modify the CI so that I can run it on this PR (instead of merging and waiting for the scheduled run)
- [x] Need to fix the command in doc string for test_strings.py
- [ ] Need to explain the `VispyTiledLayer` import in test_strings.py

### CODE REVIEW

**Comment by kcpevey** [comment]
This ran the test_translations workflow successfully over on my fork [here](https://github.com/kcpevey/napari/actions/runs/4185760511/jobs/7253276603)

**Comment by goanpeca** [comment]
Awesome @kcpevey, thanks for working on this, it had not been running for some time now!

File: napari/components/camera.py [hunk 1]
```diff
@@ -38,8 +38,8 @@ class Camera(EventedModel):
     interactive: bool = True
 
     # validators
-    @validator('center', 'angles', pre=True)
-    def _ensure_3_tuple(v):
+    @validator('center', 'angles', pre=True, allow_reuse=True)
```
Comment (goanpeca) [inline napari/components/camera.py:41]: There are a bunch of changes on this file, not sure if this is ruff related or ... 

Not sure how they relate to the translation checks 🤔

**Comment by kcpevey** [comment]
Anyone know why this one test is failing on a timeout? [ubuntu-18.04 3.8 pyqt5 min_req](https://github.com/napari/napari/actions/runs/4185980715/jobs/7255303307#logs)

File: napari/components/camera.py (hunk 1 above)
Comment (kcpevey) [reply napari/components/camera.py:41]: Good question... There are a bunch of methods that aren't being used anywhere. If they were being used, I think they would have been picked up by pydantic?

 At any rate, the translator check has a function which looks though all the methods via importing (as far as I can tell), `import_module_by_path`. When every method is imported in this way, pydantic checks are triggered. I had to make these changes to allow pydantic to pass in order to allow the `test_strings.py` file to run. 

I believe the `allow_reuse=True` method was added around 2 years ago which is when this script started failing so my guess is that when this was first set up, it wasn't required.

**Comment by psobolewskiPhD** [comment]
> Anyone know why this one test is failing on a timeout? [ubuntu-18.04 3.8 pyqt5 min_req](https://github.com/napari/napari/actions/runs/4185980715/jobs/7255303307#logs)

It should be fixed because https://github.com/napari/napari/pull/5560 got merged.
Try merging main?

**Review by goanpeca — APPROVED** [review]

**Review by Carreau — APPROVED** [review]

**Pull request merged**